src/main/java/com/hazelcast/eureka/one/EurekaOneDiscoveryStrategy.java

```diff
@@ -389,19 +393,28 @@ public boolean shouldRegisterWithEureka() {
         private final EurekaInstanceConfig instanceConfig;
         private final DiscoveryNode localNode;
         private final String uuid;
+        private final String appname;
 
         private DelegatingInstanceConfig(EurekaInstanceConfig instanceConfig, DiscoveryNode localNode) {
             this.instanceConfig = instanceConfig;
             this.localNode = localNode;
             this.uuid = UuidUtil.newSecureUuidString();
+            this.appname = instanceConfig.getAppname();
```

Comment:
Super minor: you could have the following line instead of these 4.
```java
this(instanceConfig, localNode, instanceConfig.getAppname());
```

Reply:
good spot, thanks